Header header
PhaseSpaceMarker[] markers
PhaseSpaceBody[] bodies
int32 frameNum